psl/psl-core/src/common/preview_features.rs

```diff
@@ -121,7 +120,7 @@ pub const ALL_PREVIEW_FEATURES: FeatureMap = FeatureMap {
         | ImprovedQueryRaw
         | DataProxy
         | InteractiveTransactions
-        | ClientExtensions
+        | OrderByNulls
```

Comment:
```suggestion
        | ClientExtensions
        | OrderByNulls
```

Reply:
Any reason to do this? It doesn't change anything

Reply:
I see. I wanted to look at this after https://github.com/prisma/prisma-engines/pull/3995 because we can only work on one at a time (or we have conflicts once one is merged)

Reply:
It should make the tests pass, so the PR can actually be reviewed in its current state.